### PR TITLE
deps: support django 4.1

### PR DESCRIPTION
**Pull request by csrftoken**

### CODE REVIEW

**Comment by benedikt-bartscher** [comment]
https://github.com/celery/django-celery-beat/pull/567

**Comment by evanlesmez** [comment]
How is this going?

File: requirements/docs.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Django>=2.2,<4.1
+Django>=2.2,<=4.1
```
Comment (auvipy) [inline requirements/docs.txt:1]:
```suggestion
Django>=2.2,<4.2
```

**Review by auvipy — CHANGES_REQUESTED** [review]

File: requirements/runtime.txt [hunk 2]
```diff
@@ -1 +1 @@
-Django>=2.2,<4.1
+Django>=2.2,<=4.1
```
Comment (auvipy) [inline requirements/runtime.txt:1]:
```suggestion
Django>=2.2,<4.2
```

**Review by auvipy — CHANGES_REQUESTED** [review]

File: requirements/test-django.txt [hunk 3]
```diff
@@ -1 +1 @@
-Django>=3.2,<4.1
+Django>=3.2,<=4.1
```
Comment (auvipy) [inline requirements/test-django.txt:1]:
```suggestion
Django>=3.2,<4.2
```

**Review by auvipy — CHANGES_REQUESTED** [review]

**Comment by auvipy** [comment]
not sure why py3.7 is failing

**Comment by goatwu1993** [comment]
@auvipy Upstream error by kombu & celery, caused by using importlib-metadata API which is deprecated at 5.0.0

Both libs have been fixed and merged but in 5.3.0 milestone.

https://github.com/celery/kombu/pull/1601
https://github.com/celery/celery/pull/7785

Only python < 3.8 needs importlib-metadata, which is why only python37 breaks here.

**Comment by goatwu1993** [comment]
rebase should pass 3.7

**Pull request merged**